src/main/resources/anniv5Resources/summaries/ratings.txt

```diff
@@ -13,6 +13,7 @@ BlacksmithsPack	3	3	4	4	4
 BoardGamePack	4	3	2	4	2
 BulwarkPack	2	5	1	4	4
 ClawPack	5	2	1	2	5
+ColorlessPack	4	3	3	2	5
```

Comment:
I'm a bit doubtful of the 5 in scaling here. The pack looks to have decent scaling, but it's not as straightforward as other packs with a 5 in scaling like Claw, Distortion, or Poison. For what it's worth, having played with your Secret Level pack, that 5 in scaling is probably also too high. That pack provides reasonably good scaling but it doesn't outclass the various other packs that have good scaling and are rated as a 4.

This might come from a bit of overestimating the value of the scaling effects you've created. A couple sources of Strength or Dexterity gain alongside a few other ways to scale (e.g. Prism for this pack, The Donut for Secret Level) is good and will make a pack above average, but it doesn't reach the same consistency and speed as most of the packs rated 5 in scaling.

Let's change this to a 4 in scaling and also change Secret Level to have a 4 in scaling.

Reply:
Done!